app/src/renderer/routes.js

```diff
@@ -35,12 +35,13 @@ export default [
     path: `/staking`,
     name: `Staking`,
     component: require(`./components/staking/PageStaking`).default,
-    redirect: `/staking/my-delegations/`,
+    redirect: `/staking/validators/`,
```

Comment:
this also should not be the case if the user is signed in

Reply:
done